src/lib.rs

```diff
@@ -522,6 +528,7 @@ where
 
 /// Node of the inner radix trie.
 #[derive(Clone, Debug, PartialEq, Eq)]
+#[cfg_attr(feature = "enable-serde", derive(Serialize, Deserialize))]
 pub struct IpTrieNode {
```

Comment:
I think `#[serde(transparent)]` will help output more compact result.

Reply:
This not compile.

Reply:
You can do it like 
```rust
#[cfg_attr(
    feature = "enable-serde",
    derive(Serialize, Deserialize),
    serde(transparent)
)]
```